pkg/image/origins.go

```diff
@@ -150,6 +150,7 @@ var OriginMap = map[string]string{
 	"mirrored-kiwigrid-k8s-sidecar":                           "https://github.com/kiwigrid/k8s-sidecar",
 	"mirrored-kube-rbac-proxy":                                "https://github.com/brancz/kube-rbac-proxy",
 	"mirrored-kube-state-metrics-kube-state-metrics":          "https://github.com/kubernetes/kube-state-metrics",
+	"mirrored-kube-vip-kube-vip-iptables":                     "https://github.com/harvester/harvester",
```

Comment:
Is Harvester truly the upstream for this image? I know Harvester is using it, but it is not the origin. Shouldn't it be:
```suggestion
	"mirrored-kube-vip-kube-vip-iptables":                     "https://github.com/kube-vip/kube-vip",
```

Reply:
fixed.